src/Parser/Daedalus.g4

```diff
@@ -49,7 +49,7 @@ classDef: Class nameNode '{' ( varDecl ';' )*? '}';
 prototypeDef: Prototype nameNode '(' parentReference ')' statementBlock;
 instanceDef: Instance nameNode '(' parentReference ')' statementBlock;
 instanceDecl: Instance nameNode ( ',' nameNode )*? '(' parentReference ')';
-varDecl: Var typeReference (varValueDecl | varArrayDecl) (',' (varValueDecl | varArrayDecl) )* ;
+varDecl: Var typeReference (varValueDecl | varArrayDecl) (',' (varDecl | varValueDecl | varArrayDecl) )* ;
```

Comment:
this change make it possible to declare variables like that:
```var int x, var int y;```
is it really possible in Daedalus?  (cannot check myself now)
It looks very wrong and even if it's supported in original compiler I don't think we want to have that.
Current grammar supports ```var int x; var int y;```
and ```var int x, y;```

Reply:
`var int x, var int y;` is (sadly) valid Daedalus code and in use by LeGo.